Datamigratie.Data/DatamigratieDbContext.cs

```diff
@@ -52,8 +52,42 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             entity.HasIndex(e => e.Status);
             entity.HasIndex(e => e.CreatedAt);
         });
+
+        modelBuilder.Entity<MigrationRecord>(entity =>
+        {
+            entity.HasKey(e => e.Id);
+            
+            entity.Property(e => e.DetZaaknummer)
+                .IsRequired()
+                .HasMaxLength(100);
+            
+            entity.Property(e => e.OzZaaknummer)
+                .HasMaxLength(100);
+            
+            entity.Property(e => e.IsSuccessful)
+                .IsRequired();
+            
+            entity.Property(e => e.ErrorTitle)
+                .HasMaxLength(500);
+            
+            entity.Property(e => e.ErrorDetails)
+                .HasMaxLength(2000);
+            
+            entity.Property(e => e.ProcessedAt)
+                .IsRequired();
+            
+            entity.HasOne(mr => mr.Migration)
```

Comment:
Don't think its necessary to set the foreignkey yourself. entityframework will figure this out if you model the object like this:

```
    public int MigrationId { get; set; }      
    public required Migration Migration { get; set; } 
```